users_permissions/README.md

```diff
@@ -75,59 +75,45 @@ Then run this command:
 1. Create a new app following the instructions here: [Create an App with Atlas App Services UI](https://www.mongodb.com/docs/atlas/app-services/manage-apps/create/create-with-realm-ui).
     For the purpose of this sample you don't need to create an app from a template. You can just create an empty application.
 1. Click the button `Review draft & deploy`.
-1. Go to the `Authentication` menu in the left panel and select :
+1. Go to the **Authentication** menu in the left panel and select :
     1. `Authentication Providers`. Then make sure the option "Email/Password" is ON. Save and then click the button `Review draft & deploy`. Read [this page](https://www.mongodb.com/docs/atlas/app-services/authentication/providers/) for more information about the other types of authentication.
     1. `Custom User Data`. Then make sure the option is enabled. Select cluster, database and collection where to store the custom data. Then select the field that to be used for mapping users with their custom data. For this sample the field name is `owner_id`. Read [this page](https://www.mongodb.com/docs/atlas/app-services/users/enable-custom-user-data/) for more information about enabling custom user data.
-1. Go to the `Sync` menu and [Enable Flexible Sync](https://www.mongodb.com/docs/atlas/app-services/sync/configure/enable-sync/#enable-flexible-sync).
+1. Go to the **Rules** menu.
+    1. Select "Default roles and filters" under the service name. Choose from the list with `Other preset`: `readOwnWriteOwn (User can only read and write their own data)` and click the button `Add preset role`. If you switch to `Advanced view` the json should looks like the one in this file: /assets/atlas_app/data_sources/mongodb-atlas/default_rule.json.
```

Comment:
```suggestion
    1. Select "Default roles and filters" under the service name. Choose from the list with `Other preset`: `readOwnWriteOwn (User can only read and write their own data)` and click the button `Add preset role`. If you switch to `Advanced view` the JSON should looks like the one in the file `/assets/atlas_app/data_sources/mongodb-atlas/default_rule.json`.
```

to clarify, where's the file /assets/atlas_app/data_sources/mongodb-atlas/default_rule.json?

Reply:
https://github.com/realm/realm-dart-samples/pull/44/files#diff-1315a12f811dda393587b32c9158af75558e19d9391a9cfa5c52b42ed9279385:~:text=dart_flexible_sync/atlas_app/data_sources/mongodb%2Datlas/default_rule.json